CI/PHP-CS-Fixer/example/example_cleaned.php

```diff
@@ -27,16 +28,19 @@ public function sampleMethod($a, $b = null)
 
 /** thx to https://mlocati.github.io/php-cs-fixer-configurator for the examples **/
 // cast_spaces
-$a = 0; $b = 0; $c = 0; $d = 0; $e = 0 ; $f = 0;
-$bar = (string) $a;
-$foo = (int) $b;
+$a    = 0; $b    = 0; $c    = 0; $d   =0; $e   = 0 ; $f    =0;
+$bar  = (string) $a;
+$foo  =  (int) $b;
```

Comment:
@gvollbach 
I am confused, the example now alignes the operators, but I thought we do not want that.

Reply:
The option is already removed from the config, but the example files are not cleaned, I will take care of this if all options are fixed.